src/sentry/api/issue_search.py

```diff
@@ -92,6 +94,27 @@ def convert_status_value(value, projects, user, environments):
     return parsed
 
 
+def convert_category_value(value, projects, user, environments):
+    if features.has("projects:performance-issue-details-backend", projects[0]):
+        for category in value:
```

Comment:
Since we can have categories passed here (`category:[errors, performance]`), we should get the issue types for each category and return the whole list. Right now we're only doing so for the first category.

Will be helpful in the future when we have >2 categories.